src/feedback/vue/submitFeedback.vue

```diff
@@ -15,33 +24,27 @@
         <h3 id="feedback-heading">Feedback</h3>
 
         <div>
-            <textarea style="width: 100%" rows="10" cols="90" v-model="comment"></textarea>
             <div>
-                <div
-                    style="float: right; font-size: smaller"
-                    id="remaining"
-                    :class="comment.length > 2000 ? 'exceeds-max-length' : ''"
-                >
-                    {{ 2000 - comment.length }} characters remaining
-                </div>
-                <div>
-                    <em
-                        >Feedback is <b>anonymous</b>. If you need a reply,
-                        <a href="https://github.com/aws/aws-toolkit-vscode/issues/new/choose">contact us on GitHub</a
-                        >.</em
+                <div style="margin-bottom: 10px">
+                    <div
+                        style="float: right; font-size: smaller"
+                        id="remaining"
+                        :class="comment.length > 2000 ? 'exceeds-max-length' : ''"
                     >
+                        {{ 2000 - comment.length }} characters remaining
+                    </div>
+                    <div>
+                        <em
+                            >Feedback is <b>anonymous</b>. If you need a reply,
```

Comment:
Is it auto-formatting this or something? Having the tag close on the next line with nothing in between seems unconventional.

Reply:
prettier is doing this